hack/lib/version.sh

```diff
@@ -133,8 +133,8 @@ kube::version::ldflag() {
   local val=${2}
```

Comment:
maybe double-quote these, too?

Reply:
you mean change it like this ?
`local val="${2}"`

I tested it, it didn't make any change of the value here ...

Seems only use single-quote to qoute the whole 'importpath.name=value' like:
`-X 'k8s.io/kubernetes/pkg/version.gitTreeState=git archive'` 
or only qoute the value like:
`-X k8s.io/kubernetes/pkg/version.gitTreeState='git archive'` 
could solve the problem of the value which included spaces.

Reply:
yeah, I think you're right - this PR is sufficient as-is. Thanks!